components/resources/library/src/commonMain/kotlin/org/jetbrains/compose/resources/ResourceContentHash.kt

```diff
@@ -0,0 +1,5 @@
+package org.jetbrains.compose.resources
+
+@Suppress("unused")
+@Retention(AnnotationRetention.BINARY)
+annotation class ResourceContentHash(val hash: Int)
```

Comment:
I wonder if we might want to use Long instead?

Reply:
UPD: I see that this uses String.hashCode therefore Int is used here, as well. 
Probably good enough to get started, but we should be clear about the fact that it _can_ happen that hashes collide.

Reply:
I wanted to keep the things simple. I don't know collisions probability of String.hashCode(), but I guess it is good enough for our case.  Though, no problem to make it 64-bit and more collision resistant